### DISCUSSION TITLE
skoala/tests/gateway-stress-test

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2603] [Announcements]
# skoala/tests/gateway-stress-test

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/tests/gateway-stress-test.html

**Comment by Ye-Ting** [comment]
测试报告没有输出 基础的硬件情况，没有对应的对标的分析。
